industrial_ci/src/tests/source_tests.sh

```diff
@@ -149,6 +149,14 @@ set +o pipefail
 
 ici_time_end  # rosdep_install
 
+if [ "$CATKIN_LINT" == "true" ]; then
+    ici_time_start catkin_lint
+    sudo apt-get install -qq -y python-pip
```

Comment:
`python-pip` should already be installed in our images.
I am undecided if we want to enforce it here again.